src/System.ComponentModel.TypeConverter/src/System/Drawing/PointConverter.cs

```diff
@@ -0,0 +1,162 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Collections;
+using System.ComponentModel;
+using System.Globalization;
+
+namespace System.Drawing
+{
+    public class PointConverter : TypeConverter
+    {
+        public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceType)
+        {
+            if (sourceType == typeof(string))
+            {
+                return true;
+            }
+            return base.CanConvertFrom(context, sourceType);
+        }
+
+        public override bool CanConvertTo(ITypeDescriptorContext context, Type destinationType)
+        {
+            //if (destinationType == typeof(InstanceDescriptor))
+            //{
+            //    return true;
+            //}
+            return base.CanConvertTo(context, destinationType);
+        }
+
+        [SuppressMessage("Microsoft.Performance", "CA1808:AvoidCallsThatBoxValueTypes")]
+        public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
+        {
+
+            string strValue = value as string;
+
+            if (strValue != null)
+            {
+
+                string text = strValue.Trim();
+
+                if (text.Length == 0)
+                {
+                    return null;
+                }
+                else
+                {
+
+                    // Parse 2 integer values.
+                    //
+                    if (culture == null)
+                    {
+                        culture = CultureInfo.CurrentCulture;
+                    }
+                    char sep = culture.TextInfo.ListSeparator[0];
+                    string[] tokens = text.Split(sep);
+                    int[] values = new int[tokens.Length];
+                    TypeConverter intConverter = TypeDescriptor.GetConverter(typeof(int));
+                    for (int i = 0; i < values.Length; i++)
+                    {
+                        // Note: ConvertFromString will raise exception if value cannot be converted.
```

Comment:
I guess this answers my earlier question about whether exceptions here are desired?